sonic_platform_base/sonic_sfp/sfputilbase.py

```diff
@@ -25,6 +25,8 @@
     from .sff8436 import sff8436InterfaceId  # Dot module supports both Python 2 and Python 3 using explicit relative import methods
     from .sff8436 import sff8436Dom    # Dot module supports both Python 2 and Python 3 using explicit relative import methods
     from .inf8628 import inf8628InterfaceId    # Dot module supports both Python 2 and Python 3 using explicit relative import methods
+    import sys
+    import sonic_platform_base.sonic_sfp.ext_media_api as ext_media_module
```

Comment:
Move this import to the third-party import group (the second group), and place it in alphabetical order within that group.